sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/DateTimeUtils.scala

```diff
@@ -45,6 +45,7 @@ object DateTimeUtils {
   // it's 2440587.5, rounding up to compatible with Hive
   final val JULIAN_DAY_OF_EPOCH = 2440588
   final val SECONDS_PER_DAY = 60 * 60 * 24L
+  final val SECONDS_PER_MONTH = 60 * 60 * 24 * 31D
```

Comment:
Thank you for working on this, @mgaido91 !
BTW, is the naming, `SECONDS_PER_MONTH`, correct for February?

Reply:
hi @dongjoon-hyun! thanks for the comment. Of course it is not, strictly speaking, but this is how both Spark before the PR (notice the 31.0) and Hive (which Spark draws his inspiration from for this function) work. They consider every month as having 31 days, even though this is not true of course.

Reply:
I see now your point, yes, I will inline the value as Hive does with a comment then, thanks.